packages/seo-store/src/analysis/tests.js

```diff
@@ -0,0 +1,20 @@
+import * as configSlice from "./slice/config";
```

Comment:
Instead of the line 4 destructuring, you can destructure right away here?
```suggestion
import configReducer, { initialState, configActions } from "./slice/config";
```